### ISSUE TITLE
Neo4j 4.0.0 ~ Cypher-Shell: can't create / drop database

### ISSUE THREAD

**Issue by mikeblum** [neo4j/neo4j#12403]
In the neo4j browser I can create a database but in cypher-shell it throws an error and after upgrading from the release candidate to 4.0.0 I can no longer create and destroy databases.

**Neo4j Version:** 4.0.0 Enterprise
**Operating System:** Ubuntu 18.04 
**API:** Docker + Cypher-Shell 4.0.0

### Steps to reproduce

1. Create a database against the `system` database:
```
echo 'CREATE DATABASE `localhost-test`;' | ./bin/cypher-shell --database system
```

> There is no procedure with the name `dbms.showCurrentUser` registered for this database instance. Please ensure you've spelled the procedure name correctly and that the procedure is properly deployed.

2. Destroy the database:
```
echo 'DROP DATABASE `localhost-test`;' | ./bin/cypher-shell --database system
```

> There is no procedure with the name `dbms.showCurrentUser` registered for this database instance. Please ensure you've spelled the procedure name correctly and that the procedure is properly deployed.

### Expected behavior

Database should be successfully created and the database should then be destroyed.

### Actual behavior

```
There is no procedure with the name `dbms.showCurrentUser` registered for this database instance. Please ensure you've spelled the procedure name correctly and that the procedure is properly deployed.
```

but appears to work in the browser:

![image](https://user-images.githubusercontent.com/3905463/74872419-cce87000-5322-11ea-882d-72f884b71818.png)

here is the list of procedures:

```
CALL dbms.procedures() YIELD name, signature RETURN name;
```

[records.txt](https://github.com/neo4j/neo4j/files/4227272/records.txt)

---

This was working using the previous release candidate:

```
- FROM neo4j/neo4j-experimental:4.0.0-beta03mr03-enterprise as neo4j
+ FROM neo4j:4.0.0-enterprise as neo4j
```

**Comment by mikeblum** [comment]
This appears to have been changed in the 4.1 branch of cypher-shell: https://github.com/neo4j/cypher-shell/commit/299ee388708ce1e5a0fd923e4ce3976acd0a4547

**Comment by sherfert** [comment]
It has also changed in the 4.0 branch: https://github.com/neo4j/cypher-shell/commit/665fd669cf370ae62ada38658b6d2bd7d7778090

This fix will be included in the next patch release of neo4j 4.0
